src/pymatgen/io/vasp/outputs.py

```diff
@@ -1718,11 +1719,11 @@ def __init__(
                     if tag == "generator":
                         self.generator = self._parse_params(elem)
```

Comment:
Here is a well hidden bug, after we cast keys to upper case, we cannot use `Incar` to pack "generator" tags anymore, which is not INCAR tags:
```
{'program': 'vasp', 'version': '5.4.4.18Apr17-6-g9f103f2a35', 'subversion': '(build Apr 25 2017 00:43:00) complex            parallel', 'platform': 'LinuxIFC', 'date': '2017 09 29', 'time': '11:34:32'}
```